CONTRIBUTING.md

```diff
@@ -222,25 +225,27 @@ $ git tag -a -m "Release X.Y.Z" X.Y.Z
 
 ### Update version info back to dev
 
-After tagging, the version and stability constants in `php_phongo.h` should be
+After tagging, the version and stability constants in `phongo_version.h` should be
 updated back to development status.
 
 ```
 #define PHP_MONGODB_VERSION "1.1.8"
 #define PHP_MONGODB_STABILITY "stable"
+#define PHP_MONGODB_VERSION_DESC 1,1,8,1
 ```
 
 The above would be changed to:
 
 ```
 #define PHP_MONGODB_VERSION "1.1.9-dev"
 #define PHP_MONGODB_STABILITY "devel"
+#define PHP_MONGODB_VERSION_DESC 1,1,9,0
```

Comment:
How do we increment this if going from 1.4.0-beta1 back to 1.4.0-dev and later to 1.4.0-rc1? I assume the value might transition from the following states:

 * `1,4,0,0` (original 1.4.0-dev)
 * `1,4,0,1` (first beta)
 * `1,4,0,2` (reverted back to dev)
 * `1,4,0,3` (first RC)

If so, can we summarize this in the "**Note:** If this is an alpha, beta, or RC release..." paragraph below?